ydb/core/kqp/host/kqp_host.cpp

```diff
@@ -342,8 +343,18 @@ class TAsyncPrepareYqlResult : public TKqpAsyncResultBase<IKqpHost::TQueryResult
         prepareResult.QueryAst = prepareResult.PreparingQuery->GetPhysicalQuery().GetQueryAst();
     }
 
+    void FillPartialResult(TResult& prepareResult) const override {
+        YQL_ENSURE(QueryCtx->PrepareOnly);
+
+        if (auto exprRoot = GetExprRoot()) {
+            prepareResult.PreparingQuery = std::move(QueryCtx->PreparingQuery);
+            prepareResult.PreparingQuery->MutablePhysicalQuery()->SetQueryAst(KqpExprToPrettyString(*GetExprRoot(), ExprCtx));
```

Comment:
Here is this file in FillResult there is line: 
`prepareResult.QueryAst = prepareResult.PreparingQuery->GetPhysicalQuery().GetQueryAst();`
Why don't we do the same in this function?